sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_form_training_client_async.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         )
 
     @distributed_trace_async
-    async def training(
+    async def train_model(
```

Comment:
I think we want sync and async to be named thee same, can we make the sync begin_train_model? Can't remember if we talked about this being valid before

Reply:
will make this change, thanks for pointing out!